src/holofuel/pages/CreateOfferRequest/CreateOfferRequest.js

```diff
@@ -139,12 +141,12 @@ export default function CreateOfferRequest ({ history: { push } }) {
       <h4 styleName='amount-label'>
         {title}
       </h4>
-      <div styleName='amount'>
+      <div styleName='amount' onClick={() => setNumpadVisible(true)}>
```

Comment:
Nice addition.  Did you met with C to determine whether this should be accompanied with a icon to signal this functionality?

Reply:
no this is more like an easter egg. It just seemed like the right thing to do. There could definitely be some kind of design element to point people to this feature.